doc/1/controllers/collection/list/index.md

```diff
@@ -0,0 +1,52 @@
+---
+code: true
+type: page
+title: ListAsync
+description: Returns the collection list of an index.
+---
+
+# ListAsync
+
+Returns the list of collections associated to a provided index.
+
+The returned list is sorted in alphanumerical order.
+The `from` and `size` arguments allow pagination. They are returned in the response if provided.
+
+
+## Arguments
+
+```csharp
+public async Task<JObject> ListAsync(
+      string index,
+      int? from = null,
+      int? size = null,
+      string type = null
```

Comment:
If the default value is "all", shouldn't this be `string type = "all"`?

(also shouldn't we create an enum for that one?)